hw/misc/hermes.c

```diff
@@ -142,15 +145,16 @@ struct hermes_bar2_dir_reg {
                         dma_addr_t len);
     int irq_vector;
     void *(*entry)(void*);
-    const char *name;
+    char *name;
+    size_t ch;
```

Comment:
size_t is excessive. int or unsigned int is more than enough.

Reply:
I agree. It's habit to use the size_t type when I'm in doubt.
I would say that we can even go as far as use a `unsigned char` for now.